fdbrpc/FlowTransport.actor.cpp

```diff
@@ -483,6 +487,17 @@ struct Peer : NonCopyable {
 					TraceEvent(ok ? SevInfo : SevWarnAlways, "IncompatibleConnectionClosed", conn ? conn->getDebugID() : UID()).error(e, true).suppressFor(1.0).detail("PeerAddr", self->destination);
 				}
 
+				if(self->destination.isPublic() && IFailureMonitor::failureMonitor().getState(self->destination).isAvailable()) {
```

Comment:
To make sure I understand this correctly, is it saying that we trigger the warning if we have a 20 second span where there is never a gap larger than 5 seconds between connection closed events to a particular destination?

Reply:
Yes, that is correct